src/Core/Layer/Layer.js

```diff
@@ -175,4 +182,77 @@ const ImageryLayers = {
     },
 };
 
+/**
+ * Interface implemented by {@link GeometryLayer} instances capable of displaying color layers
+ * @interface
+ */
+function CanDisplayColorLayer() {}
+/**
+ * Add a color layer to display on this geometry
+ *
+ * @function
+ * @name CanDisplayColorLayer#addColorLayer
+ * @param {Object} colorLayer - property bag defining the layer to display.
+ * @param {string} colorLayer.id - unique identifier of this layer
+ * @param {string} colorLayer.protocol - one of wms, wmts, wmtsc, tms, static.
+ * Each protocol has a set of specific options that are documented in XXX.
```

Comment:
"in XXX"?

Reply:
They're not documented yet (not in master nor in this PR)